services/wfm/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/create/FlowCreateFsm.java

```diff
@@ -92,6 +92,7 @@
     // The amount of flow create operation retries left: that means how many retries may be executed.
     // NB: it differs from command execution retries amount.
     private int remainRetries;
+    private boolean timedOut;
```

Comment:
As far as I see it's always True or undefined. Maybe we need `timeout=false` in constructor?